packages/core/lib/sandbox.js

```diff
@@ -45,23 +34,6 @@ global.WebSocketEventTarget = () => {
   throw new Error('WebSocketEventTarget is not available in the sandbox');
 };
 
-// Disable location and redirects
```

Comment:
is this functionality still being suppressed?

Reply:
Yes, I've added a Content-Security-Policy config in the html header.